audio-libs.json

```diff
@@ -0,0 +1,41 @@
+{
+    "name": "audio-libs",
+    "modules": [
+        {
+            "name": "mpg123",
+            "buildsystem": "autotools",
+            "sources": [{ 
+               "type": "archive",
+                "url": "https://iweb.dl.sourceforge.net/project/mpg123/mpg123/1.26.1/mpg123-1.26.1.tar.bz2",
+                "sha256": "74d6629ab7f3dd9a588b0931528ba7ecfa989a2cad6bf53ffeef9de31b0fe032"
```

Comment:
the SDK has this:
`lrwxrwxrwx 1 nfsnobody nfsnobody     19 Jun  1 21:28 /usr/lib/x86_64-linux-gnu/libmpg123.so.0 -> libmpg123.so.0.44.9`